docs/build/tools/foundry.mdx

```diff
@@ -79,7 +79,7 @@ available in the **Sapphire foundry** package.
 1. Install the **Sapphire Foundry** Soldeer package:
 
    ```shell
-   forge soldeer install @oasisprotocol-sapphire-foundry~0.1.0
+   forge soldeer install @oasisprotocol-integrations-foundry~0.1.1
```

Comment:
This is all Sapphire-specific. Integrations folder is there just to better organize the monorepo.

```suggestion
   forge soldeer install @oasisprotocol-sapphire-foundry~0.1.1
```

Reply:
Renamed to @oasisprotoco-sapphire-foundry, add new soldeer repo https://soldeer.xyz/project/@oasisprotocol-sapphire-foundry.